File: Back/ProEventos.Domain/Models/RedesSociais/RedeSocial.cs
using ProEventos.Domain.Models.Eventos;
using ProEventos.Domain.Models.Palestrantes;

namespace ProEventos.Domain.Models.RedesSociais
{
    public class RedeSocial
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string URL { get; set; }
        public int EventoId { get; set; }
        public Evento Evento { get; set; }
        public int PalstranteId { get; set; }
        public Palestrante Palestrante { get; set; }
    }
}

File: Front/ProEventos-app/src/app/interfaces/models/redesSociais/RedeSocial.ts
import { Evento } from "../eventos/Evento"
import { Palestrante } from "../palestrantes/Palestrante"

export interface RedeSocial {
  id: number
  nome?: string
  url?: string
  eventoId?: number
  evento?: Evento
  palstranteId?: number
  palestrante?: Palestrante
}
